collector/templates/probe.yaml

```diff
@@ -62,6 +62,30 @@ spec:
           mountPath: /etc/prequel/spool
         - name: shared-data
           mountPath: /data
+        ports:
+          - name: health
+            containerPort: 8081
+        startupProbe:
+          httpGet:
+            path: /startupz
+            port: health
+          periodSeconds: 2
+          failureThreshold: 60
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: health
+          periodSeconds: 5
+          timeoutSeconds: 1
+          failureThreshold: 3
+        livenessProbe:
+          httpGet:
+            path: /livez
+            port: health
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          timeoutSeconds: 1
+          failureThreshold: 3
```

Comment:
A few questions:
1. the requests are cheap, but it's additional traffic in hundreds nodes (hundreds probes) cluster at 5 sec interval?
2. in the current implementation all these probes will call Flush on NATS (cause rountrip call on NATS), that's another additional traffic.
3. should we exclude this traffic from our HTTP events monitoring? The probes are already not attaching to it's own proc in my implementation, but they will attach to the collector and k8s processes that will call these checks.

Reply:
> the requests are cheap, but it's additional traffic in hundreds nodes (hundreds probes) cluster at 5 sec interval

good point. we can bump the period to 60s

> should we exclude this traffic from our HTTP events monitoring

yeah we should probably ignore our own traffic -- pixie does something like this i think